docs/pages/base/api/use-button.json

```diff
@@ -0,0 +1,66 @@
+{
+  "inputParams": {
+    "disabled": {
+      "type": { "name": "boolean", "description": "If `true`, the component is disabled." },
+      "default": "false"
+    },
+    "focusableWhenDisabled": {
+      "type": {
+        "name": "boolean",
+        "description": "If `true`, allows a disabled button to receive focus."
+      },
+      "default": "false"
+    },
+    "href": { "type": { "name": "string", "description": "" } },
+    "onFocusVisible": { "type": { "name": "React.FocusEventHandler", "description": "" } },
+    "ref": { "type": { "name": "React.Ref<any>", "description": "" } },
+    "tabIndex": {
+      "type": { "name": "NonNullable<React.HTMLAttributes<any>['tabIndex']>", "description": "" }
+    },
+    "to": { "type": { "name": "string", "description": "" } },
+    "type": {
+      "type": {
+        "name": "React.ButtonHTMLAttributes<HTMLButtonElement>['type']",
+        "description": "Type attribute applied when the `component` is `button`."
+      },
+      "default": "'button'"
+    }
+  },
+  "returnValue": {
+    "active": {
+      "type": { "name": "boolean", "description": "If `true`, the component is active (pressed)." },
+      "default": "false",
+      "required": true
```

Comment:
This feels weird in the context of a returned value, e.g.

<img width="151" alt="Screenshot 2023-01-21 at 18 20 18" src="https://user-images.githubusercontent.com/3165635/213879009-0ecb0453-25f7-446b-9e50-327100e21b97.png">

https://deploy-preview-35828--material-ui.netlify.app/base/api/use-button/#returnValue

I think that what MUI X did with the optional values would make more sense in this context: 

<img width="222" alt="Screenshot 2023-01-21 at 18 22 22" src="https://user-images.githubusercontent.com/3165635/213879097-ea99d517-fb58-4d4e-9819-5fb613639142.png">

https://mui.com/x/api/data-grid/grid-filter-model/#properties

Reply:
Ah nice, yep it looks better, will re-work it 👍

Reply:
Done with https://github.com/mui/material-ui/pull/35828/commits/140683198cb9a5358e59b255d2a24a103a7fa3fc